ci/cfg/ci_aes_sca_random_cw305.yaml

```
@@ -24,21 +24,22 @@ waverunner:
 capture:
   scope_select: husky
   # scope_select: waverunner
-  num_traces: 500
+  num_traces: 10000
   show_plot: True
   plot_traces: 100
-  trace_db: ot_trace_library
-  # trace_db: cw
+  #trace_db: ot_trace_library
```

```suggestion
  # trace_db: ot_trace_library
```
Do be consistent with the other config files here in this directory, I would suggest keeping the whitespace.

---

SG